modules/containers/ecr/app.py

```diff
@@ -0,0 +1,57 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import aws_cdk as cdk
+from aws_cdk import CfnOutput
+
+from stack import EcrStack
+
+project_name = os.getenv("SEEDFARMER_PROJECT_NAME", "")
+deployment_name = os.getenv("SEEDFARMER_DEPLOYMENT_NAME", "")
+module_name = os.getenv("SEEDFARMER_MODULE_NAME", "")
+app_prefix = f"{project_name}-{deployment_name}-{module_name}"
+
+DEFAULT_REPOSITORY_NAME = "ecr-repository"
```

Comment:
The default name or the custom name should always be prepended with the `app_prefix` because if i deploy multiple instances of the same module with the default ECR name, it caused conflicts in my other workloads.

Reply:
Thanks, updated